cmake/Modules/FindTileDB.cmake

```diff
@@ -5,6 +5,11 @@
 #Disable Master Catalog in TileDB
 list(APPEND TILEDB_CMAKE_ARGS "-DENABLE_MASTER_CATALOG:BOOL=False")
 
+#Support for compiling with MacOS Sierra and High Sierra
+if(CMAKE_FIND_FRAMEWORK STREQUAL "LAST")
```

Comment:
```
if(CMAKE_FIND_FRAMEWORK)
    list(APPEND TILEDB_CMAKE_ARGS "-DCMAKE_FIND_FRAMEWORK:STRING=${CMAKE_FIND_FRAMEWORK}")
endif()
```

Reply:
Shouldn't be MAKE_FIND_FRAMEWORK for all the std and not only String?

Reply:
Yep, ideally. Unfortunately, when passing CMAKE_ARGS (ExternalProjectAdd directive in the file), cmake seems to require the type of variable to be specified. Quite annoying really - didn't work without type in a few settings.

Reply:
Oh, OK...